lib/memzip/README.md

```diff
@@ -0,0 +1,28 @@
+MEMZIP - a simple readonly file system
+
+memzip takes a zip file which is comprised of uncompressed files and presents
+allows python files to be imported.
```

Comment:
Error in wording.  Maybe "... and presents it as a filesystem, allowing Python files to be imported."